webapp/locales/de.json

```diff
@@ -313,12 +313,14 @@
       },
       "columns": {
         "name": "Name",
-        "slug": "Alias"
+        "slug": "Alias",
+        "unblock": "Entblocken"
       },
       "empty": "Bislang hast du niemanden blockiert.",
       "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü blockieren.",
       "block": "Nutzer blockieren",
-      "unblock": "Nutzer entblocken"
+      "unblock": "Nutzer entblocken",
+      "unblocked": "{name} ist wieder entblockt"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-04T07:20:33Z" title="Monday, November 4th 2019, 8:20:33 am +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -293,12 +293,14 @@
       },
       "columns": {
         "name": "Name",
-        "slug": "Alias"
+        "slug": "Alias",
+        "unblk": "Entblocken"
       },
       "empty": "Bislang hast du niemanden blockiert.",
       "how-to": "Du kannst andere Benutzer auf deren Profilseite über das Inhaltsmenü blockieren.",
       "block": "Nutzer blockieren",
-      "unblock": "Nutzer entblocken"
+      "unblock": "Nutzer entblocken",
+      "unblocked": " ist wieder entblockt"
```


Couldn't we just always use the `unblock` translation and remove the `unblk` one?

Also, I am not sure I like the word `Entblocken`... Maybe `Freigeben`, `Entsperren` or `Zulassen` could be better options? @Tirokk what do you think? You're good with words. :smile: